### ISSUE TITLE
emscripten and FS manual declaration

### ISSUE THREAD

**Issue by dfober** [grame-cncm/faust2webaudio#12]
I note that emscripten functions and FS are manually declared in libFaustLoader.d.ts
You could probably save time and maintenance effort by using the existing types declarations available from [npm](https://www.npmjs.com/package/@types/emscripten)

**Comment by Fr0stbyteR** [comment]
The Emscripten d.ts was not complete while I developed the project. I'm not sure if it works better now...

**Comment by dfober** [comment]
I see, @types/emscripten is already part of the package. The version has evolved (1.39.4), maybe it's worth a try.
It seems to work with FS but I don't know how to do with the LibFaust (extends EmscriptenModule).
Actually, it seems that you only declared FS, right?

**Comment by Fr0stbyteR** [comment]
Yes indeed, `LibFaust` is extended from `EmscriptenModule` from `@types/emscripten` with some additional methods and FS.

**Comment by dfober** [comment]
seems like it's not necessary any more.
I'll do some tests and I'll keep you informed.

**Comment by dfober** [comment]
Seems to work as expected. I've pushed the change on a new branch named 'dom'.
